Revisions to chapter 9 (#206)
browserengineering/book: book/scripts.md

``` {.python expected=False}
class Tab:
    def click(self, x, y):

>         while elt:

Reviewer: Perhaps this is confusing, but I usually try to add "nesting" hints like this so readers can find the right line of code. Plus it justifies indenting things the right amount.

Author: I removed it because I found the "while elt" confusing when reading the chapter, as there is no indication of why it's there yet. I think it will be easy to find the code because it's in the same method.

Reviewer: OK. Can we un-indent the later lines of code? (Is the differ unhappy with that?)

        self.document.paint(self.display_list)
```

[^why-no-draw]: Why is `Browser.draw` not one of the phases that has to run?
After all, running `paint` will have no visible effect until `draw` happens.
It's technically not necessary right now because the only way to cause a script
to run is loading or event handling, and you'll see that in  all such cases,

> such as `handle_key`, `draw` is called at the end. (All of this will be cleaned
> up and made more precise in [Chapter 13](scheduling-and-threading.md).)

Reviewer: ```suggestion
such as `handle_key`, `draw` is called at the end. (Subtleties like this will be
clarified and made more precise in [Chapter 13](scheduling-and-threading.md).)
```

    `link` nodes are created. This is tricky to get right, so I'm
    skipping it here.

> Now, whenever the page changes, we can update its rendering again by calling

Author: Doesn't it need to also call draw? I think the browser has a bug in that render happens but not draw in this case.

Author: I answered my own question and added a footnote about it.
